lib/AST/ASTDumper.cpp

```diff
@@ -1834,9 +1834,17 @@ class PrintExpr : public ExprVisitor<PrintExpr> {
 
   void visitAdjointExpr(AdjointExpr *E) {
     printCommon(E, "adjoint_expr");
-    OS << " original=";
-    E->getOriginalExpr()->dump(OS);
-    OS << ')';
+    PrintWithColorRAII(OS, TypeReprColor) << " base_type='";
+    if (auto baseRepr = E->getBaseType().getTypeRepr())
+      baseRepr->print(PrintWithColorRAII(OS, TypeReprColor).getOS());
```

Comment:
write: "auto *baseRepr" for consistency with existing code

Reply:
Done.